components/SearchBar.tsx

```
@@ -0,0 +1,60 @@
+        value={searchTerm}
+        onChange={handleSearch}
+        placeholder="Search articles..."
+        className="w-full pl-10 pr-4 py-2 border border-gray-300 rounded-lg focus:outline-none focus:ring-2 focus:ring-blue-500"
+      />
+      <Search className="absolute left-3 top-2.5 h-5 w-5 text-gray-400" />
+      
+      {/* Results without proper keys */}
+      {results.length > 0 && (
+        <div className="absolute top-full left-0 right-0 bg-white border border-gray-200 rounded-lg shadow-lg mt-1 max-h-60 overflow-y-auto">
+          {results.map((result) => (
+            <div className="p-3 hover:bg-gray-50 cursor-pointer border-b last:border-b-0">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)